spec/frameSpec.js

```diff
@@ -0,0 +1,69 @@
+describe('Frame', function(){
+
+  var frame;
+   beforeEach(function(){
+    frame = new Frame();
+  });
+
+  describe('returns the number of pins standing', function(){
+    
+    it('after one roll', function(){
+      frame.roll(3);
+      expect(frame.pins).toEqual(7);
+      frame = new Frame();
+      frame.roll(6);
+      expect(frame.pins).toEqual(4);
+    });
+
+    it('after two rolls', function(){
+      frame.roll(4);
+      frame.roll(5);
+      expect(frame.pins).toEqual(1);
+      frame = new Frame();
+      frame.roll(3);
+      frame.roll(2);
+      expect(frame.pins).toEqual(5);
+    });
+  });
+
+  describe('is over if there has been', function(){
+
+    it('two rolls', function(){
+      expect(frame.is_over).toEqual(false);
+      frame.roll(3);
+      frame.roll(5);
+      expect(frame.is_over).toEqual(true);
+    });
+
+    it('a strike', function(){
+      expect(frame.is_over).toEqual(false);
```

Comment:
Extract this into its own test - then you can assume that if it passes, it will be the correct state for future tests